sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/EventTimeWatermarkExec.scala

```diff
@@ -81,7 +81,7 @@ class EventTimeStatsAccum(protected var currentStats: EventTimeStats = EventTime
 case class EventTimeWatermarkExec(
     eventTime: Attribute,
     delay: CalendarInterval,
-    child: SparkPlan) extends SparkPlan {
+    child: SparkPlan) extends UnaryExecNode {
```

Comment:
Hm, I checked the Spark code style guide and it doesn't say anything about `override`. Then this is good to me.

By the way, could you spend one minute creating a JIRA ticket for this PR since it changes codes, please?